config/rootcompute/kube-1.24/clusterrole-kubernetes-maximal-permission-policy.yaml

```diff
@@ -26,3 +26,6 @@ rules:
 - apiGroups: ["networking.k8s.io"]
   resources: ["ingresses/status"]
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["pods", "pods/status"]
```

Comment:
no logs, exec, ... ?

Reply:
yes, added all the supported subresources.